prov/verbs/src/verbs_domain.c

```diff
@@ -58,25 +55,35 @@ static void vrb_set_credit_handler(struct fid_domain *domain_fid,
 	domain->send_credits = credit_handler;
 }
 
+static int vrb_enable_ep_flow_ctrl(struct fid_ep *ep_fid)
+{
+	struct vrb_ep *ep = container_of(ep_fid, struct vrb_ep, util_ep.ep_fid);
+	// only enable if we are not using SRQ
+	if (!ep->srq_ep && ep->ibv_qp->qp_type == IBV_QPT_RC) {
```

Comment:
nit: I'd go a step further and verify that ep->ibv_qp is set prior to dereferencing it.